package.json

```diff
@@ -14,7 +14,7 @@
     "test": "jest && tsc --noEmit",
     "test:watch": "jest --watch",
     "prepare": "husky",
-    "cypress:open": "cypress open --browser electron",
+    "cypress:open": "cypress open --browser chrome",
```

Comment:
is this on purpose?

Reply:
Yeah - seems more sensible to be loading with chrome with the redux dev-tools.